execution_engine_testing/tests/src/test/regression/host_function_metrics_size_and_gas_cost.rs

```diff
@@ -0,0 +1,110 @@
+use casper_engine_test_support::{
+    utils, DeployItemBuilder, ExecuteRequestBuilder, InMemoryWasmTestBuilder, DEFAULT_ACCOUNT_ADDR,
+    DEFAULT_PAYMENT, DEFAULT_RUN_GENESIS_REQUEST,
+};
+use casper_execution_engine::core::engine_state::ExecuteRequest;
+use casper_types::{
+    account::{AccountHash, ACCOUNT_HASH_LENGTH},
+    bytesrepr::Bytes,
+    runtime_args,
+    system::standard_payment,
+    ApiError, RuntimeArgs, U512,
+};
+use rand::prelude::{Rng, SeedableRng, StdRng};
+
+const CONTRACT_HOST_FUNCTION_METRICS: &str = "host_function_metrics.wasm";
+const CONTRACT_TRANSFER_TO_ACCOUNT_U512: &str = "transfer_to_account_u512.wasm";
+
+const HOST_FUNCTION_METRICS_STANDARD_SIZE: usize = 90_620;
+const HOST_FUNCTION_METRICS_STANDARD_GAS_COST: u64 = 141_111_502_050;
+
+const HOST_FUNCTION_METRICS_MIN_SIZE: usize = HOST_FUNCTION_METRICS_STANDARD_SIZE * 95 / 100;
+const HOST_FUNCTION_METRICS_MAX_SIZE: usize = HOST_FUNCTION_METRICS_STANDARD_SIZE * 105 / 100;
+const HOST_FUNCTION_METRICS_MIN_GAS_COST: u64 = HOST_FUNCTION_METRICS_STANDARD_GAS_COST * 95 / 100;
+const HOST_FUNCTION_METRICS_MAX_GAS_COST: u64 = HOST_FUNCTION_METRICS_STANDARD_GAS_COST * 105 / 100;
+
+const ACCOUNT0_ADDR: AccountHash = AccountHash::new([42; ACCOUNT_HASH_LENGTH]);
+const ACCOUNT1_ADDR: AccountHash = AccountHash::new([43; ACCOUNT_HASH_LENGTH]);
+
+#[test]
```

Comment:
Should have the `#[ignore]` attribute applied as per the other EE tests in this crate.  Same for the test below.